configure.ac

```diff
@@ -1370,6 +1457,7 @@ conftemp="${datadir}"
 eval conftemp=\"${conftemp}\"
 eval conftemp=\"${conftemp}\"
 DATADIR=${conftemp}
+NUT_DATADIR=${conftemp}
```

Comment:
typo: variable renamed to NUT_DATADIR, but the below (and probably other places) AC_DEFINE_UNQUOTED was not updated